cpp/ql/src/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -509,12 +509,15 @@ class DefinitionByReferenceNode extends InstructionNode {
  * A node representing the memory pointed to by a function argument.
  *
  * This class exists only in order to override `toString`, which would
- * otherwise be the default implementation inherited from `InstructionNode`.
+ * otherwise be the default implementation inherited from `OperandNode`.
  */
-private class ArgumentIndirectionNode extends InstructionNode {
-  override ReadSideEffectInstruction instr;
+private class ArgumentIndirectionNode extends OperandNode {
```

Comment:
It looks to me like this class can now be merged into `SideEffectArgumentNode`.

Reply:
Done in 6c87b08c69d10ecebc2889c5d4c1c4ffa09518f3.